kotlinsample/src/main/java/com/airbnb/epoxy/kotlinsample/models/ItemTitleTextView.kt

```diff
@@ -0,0 +1,30 @@
+package com.airbnb.epoxy.kotlinsample.models
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.ViewGroup
+import androidx.appcompat.widget.AppCompatTextView
+import com.airbnb.epoxy.ModelView
+import com.airbnb.epoxy.TextProp
+
+@ModelView(autoLayout = ModelView.Size.MANUAL)
+class ItemTitleTextView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+) : AppCompatTextView(context, attrs, defStyle) {
+
+    init {
+        layoutParams = ViewGroup.MarginLayoutParams(
+            ViewGroup.MarginLayoutParams.MATCH_PARENT,
+            ViewGroup.MarginLayoutParams.WRAP_CONTENT
+        ).apply {
+            setMargins(50, 50, 50, 50)
+        }
+    }
+
+    @TextProp
+    fun setTitle(title: CharSequence?) {
+        text = title
+    }
+}
```

Comment:
@dinhlamvn please add a new empty line here, it may help your PR pass the CI check :smile: